ptop/main.py

```diff
@@ -28,23 +28,127 @@ def main():
                                  simple
                                  blackonwhite
                             ''')
+
+        parser.add_argument('-cpu',
+                            dest='cpu',
+                            action='store',
+                            type=float,
+                            required=False,
+                            help=
+                            '''
+                                Input sensor
+        						update interval in
+        						milli seconds less than 1000.
+        						For example 500
+                            ''')
+
+        parser.add_argument('-disk',
+                            dest='disk',
+                            action='store',
+                            type=float,
+                            required=False,
+                            help=
+                            '''
+                                Input sensor
+        						update interval in
+        						milli seconds less than 1000.
+        						For example 500
+                            ''')
+
+        parser.add_argument('-mem',
+                            dest='mem',
+                            action='store',
+                            type=float,
+                            required=False,
+                            help=
+                            '''
+                                Input sensor
+        						update interval in
+        						milli seconds less than 1000.
+        						For example 500
+                            ''')
+
+        parser.add_argument('-proc',
+                            dest='proc',
+                            action='store',
+                            type=float,
+                            required=False,
+                            help=
+                            '''
+                                Input sensor
+        						update interval in
+        						milli seconds less than 1000.
+        						For example 500
+                            ''')
+
+        parser.add_argument('-sys',
+                            dest='sys',
+                            action='store',
+                            type=float,
+                            required=False,
+                            help=
+                            '''
+                                Input sensor
+        						update interval in
+        						milli seconds less than 1000.
+        						For example 500
+                            ''')
+        
         parser.add_argument('-v',
                             action='version',
                             version='ptop {}'.format(__version__))
 
         results = parser.parse_args()
+
         if results.theme:
             theme = results.theme
         else:
             theme = 'elegant'
+
```

Comment:
Try using ternary expressions to reduce the size of the code and also helper functions.